CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# OP-TEE - version 1.1.0
+
+
+## New features
+
+* Softfloat library: floating point support is now available in 32bits TA.
+
+* Support running 64-bits TA: on ARMv8 platform, TA can be compiled in
+  AArch32 and/or in AArch64 in case the core is compiled in AArch64.
+  An example can be found in HiKey configuration file, within the excerpt code
+
+    ta-targets = ta_arm32
```

Comment:
Doesn't seem like these two lines render correctly (look at the "rich diff"). Try add the:
\`\`\`
ta-targets ....
\`\`\`